reverse-proxy.md

```diff
@@ -257,7 +257,7 @@ server {
     server_name <your-nc-domain>;
 
     location / {
-        resolver localhost;
+        resolver localhost; # Note: if you don't run a dns resolver on localhost:53, you should use another resolver
         proxy_pass http://localhost:11000$request_uri;
```

Comment:
```suggestion
        proxy_pass http://localhost:11000$request_uri; # Node: you need to change localhost to 127.0.0.1 / [::1], if you don't use a valid dns resolver in the line above
```

Reply:
Yeah looks good but should be `Note` instead of `Node`

Reply:
Also would you need to add two lines if you use both ipv4 and ipv6?

Reply:
> Also would you need to add two lines if you use both ipv4 and ipv6?

No, but if a system runs ipv6 only it needs [::1] and ipv4 only, it needs 127.0.0.1, if both are enabled it doesn't matter what is used